py3votecore/consensus_under_deadline.py

```diff
@@ -0,0 +1,394 @@
+import doctest
+import random
+import logging
+from collections import Counter
+
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Global logging configuration should not be set in a unit - only in the main program